### DISCUSSION TITLE
Overlay for age restrictions controlled by player

### DISCUSSION THREAD

**Discussion by winklesi** [SRGSSR/pillarbox-documentation#61] [Ideas]
At Play we implement for letterbox an overlay in the player for age restrictions (like play suisse has). Could this be an improvement for Pillarbox to show this banner controlled by the player? Or is this already a requirement?
<img width="1264" alt="Bildschirmfoto 2024-01-04 um 14 12 33" src="https://github.com/SRGSSR/pillarbox-documentation/assets/46777146/ff555077-9ae7-4c19-827e-d41b17d9cd1c">

**Comment by amtins** [comment]
@winklesi sorry for the late reply, I had forgotten to subscribe to this repo's notification. That's an interesting question. We can take time to discuss it tomorrow.

**Reply by winklesi** [reply]
Hey @amtins perhaps we can discuss in the next meeting?
